src/definition/helper/userPreference.ts

```diff
@@ -0,0 +1,6 @@
+import { Language } from '../../lib/Translation/translation';
+
+export interface IPreference {
```

Comment:
we can put all these small definiations in a single File, or maybe make 2-3 files organize them based on feature like one file for auth only and one for sending mail and so on.